<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1DevControl</class>
 <widget class="QWidget" name="Tab1DevControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,1">
       <item>
        <widget class="QPushButton" name="pPBtimerStart">
         <property name="text">
          <string>TimerStart</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pCBtimerValue">
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pPBquit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDial" name="pDialLed">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>3.700000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="pLcdNumberLed">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Mode::Hex</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::SegmentStyle::Filled</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="pProgressBarLed">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pDialLed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pLcdNumberLed</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
